.circleci/config.yml

```diff
@@ -1203,9 +1203,8 @@ jobs:
           time docker pull ${COMMIT_DOCKER_IMAGE} >/dev/null
           export id=$(docker run --env-file "${BASH_ENV}" --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -t -d -w /var/lib/jenkins ${COMMIT_DOCKER_IMAGE})
 
-          export COMMAND='((echo "sudo chown -R jenkins workspace && cd workspace && . ./.circleci/scripts/python_doc_push_script.sh docs/'$target' master site") | docker exec -u jenkins -i "$id" bash) 2>&1'
+          export COMMAND='((echo "sudo chown -R jenkins workspace && cd workspace && '"export CIRCLE_SHA1='$CIRCLE_SHA1'"' && . ./.circleci/scripts/python_doc_push_script.sh docs/'$target' master site") | docker exec -u jenkins -i "$id" bash) 2>&1'
```

Comment:
Is there a difference between $CIRCLE_SHA1 and ${CIRCLE_SHA1} ?

I think this approach should fix it. I'm not a bash expert, I am worried that this might just make it print $CIRCLE_SHA1 literally

Reply:
There shouldn't be a difference; for instance, you can try this Bash snippet as a sort of "local unit test":
```bash
CIRCLE_SHA1="abcdef0123456789"
export COMMAND='((echo "sudo chown -R jenkins workspace && cd workspace && '"export CIRCLE_SHA1='$CIRCLE_SHA1'"' && . ./.circleci/scripts/python_doc_push_script.sh docs/'$target' master site") | docker exec -u jenkins -i "$id" bash) 2>&1'
echo ${COMMAND} > ./command.sh
cat command.sh
```
Output:
```
((echo "sudo chown -R jenkins workspace && cd workspace && export CIRCLE_SHA1='abcdef0123456789' && . ./.circleci/scripts/python_doc_push_script.sh docs/ master site") | docker exec -u jenkins -i "$id" bash) 2>&1
```

Reply:
The other question I would have here is: there's a lot of quoting going on so I am not completely sure if this works (again, am not a bash expert)

Reply:
true 😅 I'm guessing this'll work (given the output of the above test snippet) but I guess we'll have to see